From: Juergen Repp <juergen_repp@web.de>
Date: Fri, 10 Apr 2026 05:38:47 +0000
Subject: FAPI: Fix wrong format directive in ifap_set_auth

Origin: upstream,commit:4e2fe02fca8409f875a8693ece06b8b5e5baf5ce

%u was used for a 16 bit unsigned integer and %lu for sizeof.
---
 src/tss2-fapi/fapi_util.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/tss2-fapi/fapi_util.c b/src/tss2-fapi/fapi_util.c
index 82d0d0f..ae272b5 100644
--- a/src/tss2-fapi/fapi_util.c
+++ b/src/tss2-fapi/fapi_util.c
@@ -441,7 +441,7 @@ ifapi_set_auth(
         if (auth != NULL) {
             authValue.size = strlen(auth);
             if (authValue.size > sizeof(TPMU_HA)) {
-                return_error2(TSS2_FAPI_RC_BAD_VALUE, "Size of auth value %u > %lu",
+                return_error2(TSS2_FAPI_RC_BAD_VALUE, "Size of auth value %"PRIu16" > %zu",
                               authValue.size, sizeof(TPMU_HA));
             }
             memcpy(&authValue.buffer[0], auth, authValue.size);
